packages/manager-react-components/src/components/content/dashboard-tile/tile-block.component.tsx

```diff
@@ -3,23 +3,22 @@ import { OsdsText, OsdsDivider } from '@ovhcloud/ods-components/react';
 import { ODS_TEXT_LEVEL, ODS_TEXT_SIZE } from '@ovhcloud/ods-components';
 import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
 
-export type TileBlockProps = {
-  label: string;
-};
+export type TileBlockProps = React.PropsWithChildren<{
+  label?: string;
+}>;
 
-export const TileBlock: React.FC<React.PropsWithChildren<TileBlockProps>> = ({
-  label,
-  children,
-}) => (
+export const TileBlock: React.FC<TileBlockProps> = ({ label, children }) => (
   <div className="flex flex-col mb-3">
-    <OsdsText
-      className="mb-2"
-      size={ODS_TEXT_SIZE._200}
-      level={ODS_TEXT_LEVEL.heading}
-      color={ODS_THEME_COLOR_INTENT.text}
-    >
-      {label}
-    </OsdsText>
+    {label && (
```

Comment:
**niptick** : why not add a unit test for that condition ?

Reply:
it's good now